src/chainstate/stacks/boot/costs.clar

```diff
@@ -160,6 +160,9 @@
 (define-read-only (cost_filter (n uint))
     (runtime u1000))
 
+(define-read-only (cost_buff_to_int (n uint))
+    (runtime u1000))
+
```

Comment:
Unfortunately we can't make cost-contract changes this way -- with this change, the "costs" contract published at genesis would change, which would change the consensus value for the stacks blockchain starting at height = 0. This is basically issue #2682. For now, I'd recommend just reusing an existing cost function for the new methods, and leaving a todo comment.

Reply:
I reverted this now. The cost should now be specified in src/vm/functions/mod.rs, ClarityCostFunction::Mul.